pkg/repository/testworkflow/mongo.go

```diff
@@ -78,27 +78,34 @@ func populateConfigParams(resolvedWorkflow *testkube.TestWorkflow, configParams
 
 	for k, v := range resolvedWorkflow.Spec.Config {
```

Comment:
Is this function used? I don't think that Mongo repository should ever receive the sensitive config params

Reply:
Yes, I didn't change it. It receive only non sensitive config params